src/coreclr/jit/gentree.h

```diff
@@ -556,6 +556,10 @@ enum GenTreeFlags : unsigned int
     GTF_MDARRLEN_NONFAULTING    = 0x20000000, // GT_MDARR_LENGTH -- An MD array length operation that cannot fault. Same as GT_IND_NONFAULTING.
 
     GTF_MDARRLOWERBOUND_NONFAULTING = 0x20000000, // GT_MDARR_LOWER_BOUND -- An MD array lower bound operation that cannot fault. Same as GT_IND_NONFAULTING.
+
+#ifdef TARGET_XARCH
+    GTF_HW_EM_OP                  = 0x10000000, // GT_HWINTRINSIC -- node is used as an operand to an embedded mask
```

Comment:
I tried to figure out a way to do this without a flag, but we have several intrinsics that can be contained for a multiple of reasons and so it ends up being a bit cleaner to just tag the node that is contained for embedded masking purposes to differentiate it.

After we finish plumbing through the rest of the instructions, we might find its actually not needed (we might be able to introduce helper IDs to disambiguate for example) and we can remove it then if appropriate.

Reply:
It can be, but there isn't a strong need for it, so I'd prefer to handle in a follow up to avoid the additional CI churn (will fix it in this PR if I need to push anything for the seemingly unrelated `tls_InlinedThreadStatic` failure though)